core/sdk/src/test/java/com/mongodb/stitch/core/auth/internal/CoreStitchAuthUnitTests.java

```diff
@@ -196,22 +201,105 @@ public void testLogoutBlocking() {
 
     assertFalse(auth.isLoggedIn());
 
-    auth.loginWithCredentialInternal(new AnonymousCredential());
+    final CoreStitchUser user1 =
+        auth.loginWithCredentialInternal(new AnonymousCredential());
+    final CoreStitchUser user2 =
+        auth.loginWithCredentialInternal(new UserPasswordCredential("hi", "there"));
+    final CoreStitchUser user3 =
+        auth.loginWithCredentialInternal(new UserPasswordCredential("bye", "there"));
 
+    assertEquals(auth.listUsers().getLast(), user3);
     assertTrue(auth.isLoggedIn());
 
     auth.logoutInternal();
 
+    // assert that though one user is logged out, two users are still listed
+    assertEquals(auth.listUsers().size(), 3);
```

Comment:
[task] can you add a check that the profile information is still visible for the logged out user?